imgix/urlbuilder.py

```diff
@@ -46,17 +46,24 @@ def __init__(
             use_https=True,
             sign_key=None,
             shard_strategy=SHARD_STRATEGY_CRC,
-            sign_with_library_version=True):
+            sign_with_library_version=None,
+            include_library_param=True):
+
+        if sign_with_library_version is not None:
+            warnings.warn('`sign_with_library_version` has been deprecated.' +
+                          'Use `include_library_param` instead.',
+                          DeprecationWarning, stacklevel=2)
 
         if not isinstance(domains, (list, tuple)):
             domains = [domains]
 
+        include_library = include_library_param or sign_with_library_version
```

Comment:
So there's no confusion, passing either `True` or `False` into the old `sign_with_library_version` will raise a warning as well as sign with the library version. The only way to not sign with the library version is by passing `False` into `sign_with_library_version`

Reply:
That statement seems contradictory. Is there a typo in there, or am I missing something?

Reply:
Sorry about that, I edited my comment.

Reply:
I think this is not the behavior we want. I know it makes the logic a lot more complicated to do it this way, but if we're *deprecating* this parameter but not *removing* it yet, we still need to honor its intention.

I am not a pythonista, but something like this I think?

```python
include_library = sign_with_library_version if sign_with_library_version is not None else include_library_param;
```

Reply:
That makes sense, thanks for clearing that up.